PurchasesTests/Purchasing/ProductInfoExtractorTests.swift

```diff
@@ -0,0 +1,194 @@
+import XCTest
+import Nimble
+
+import Purchases
+
+class ProductInfoExtractorTests: XCTestCase {
+
+    func testExtractInfoFromProductExtractsProductIdentifier() {
+        let product = MockSKProduct(mockProductIdentifier: "cool_product")
```

Comment:
I would extract `"cool_product"` to a `expected`,  so that you can assert against it in line 14